docs/themes/techdoc-brigade/layouts/partials/header.html

```diff
@@ -1,5 +1,5 @@
 <header>
-<section class="sticky-banner">You are viewing docs for Brigade v2. Click <a href="https://docs.brigade.sh/">here for v1 docs</a>.</section>
+<section class="sticky-banner">You are viewing docs for Brigade v2. Click <a href="https://v1--brigade-docs.netlify.app/">here for v1 docs</a>.</section>
```

Comment:
A reminder to update Netlify to deploy v1 docs to this URL.  At time of writing, it doesn't appear to be up yet.

Reply:
Thanks for the reminder. I will re-enable and force push so previews of these changes get built.

Reply:
Re-enabled.

Reply:
Can you double-check this?  Not blocking this PR as it's a Netlify task, but I'm seeing 'Not Found - Request ID: <request id>' when I go to https://v1--brigade-docs.netlify.app/